test/lit/merge/start.flip.wat.second

```diff
@@ -0,0 +1,8 @@
+(module
+  (func $start
+    ;; Not a start function, but the name overlaps so it will get deduplicated.
```

Comment:
Looks like this is getting optimized out of the merged module.

Reply:
Good point, I thought this didn't matter when I added global dce but it's best to keep the alive to see the deduplication etc.